src/lib/nodes/adm/AdmClient.ts

```diff
@@ -66,4 +70,12 @@ export class AdmClient extends Client<AdmNode> {
 
     return result.height
   }
+
+  getTimeDelta() {
+    return this.getNode().timeDelta
+  }
+
+  async sendTransaction(transaction: RegisterChatMessageTransaction): Promise<CreateNewChatMessageResponseDto> {
+    return this.post('/api/chats/process', () => ({ transaction }))
+  }
```

Comment:
We have two endpoints on https://schema.adamant.im:

- `POST /api/chats/process`
- `POST /api/transactions/process`

Better to use more descriptive method name to avoid confusion, e.g. `sendChatTransaction`